.gitignore

```diff
@@ -358,3 +358,4 @@ obj/
 !/gradle/wrapper/gradle-wrapper.jar
 
 # End of https://www.toptal.com/developers/gitignore/api/java,gradle,kotlin,android,intellij+all,androidstudio,macos,windows
+github.properties
```

Comment:
@cgedrim-co what is in your `github.properties` file and how comes it's ignored?

Reply:
It's used locally to pull packages from the github maven repo